deep_field_metadetect/gaussmom/gaussmom.py

```diff
@@ -0,0 +1,558 @@
+from dataclasses import dataclass
+
+import jax
+import jax.numpy as jnp
+import ngmix
+import numpy as np
+from jax import tree_util
+from ngmix.moments import MOMENTS_NAME_MAP
+
+from deep_field_metadetect.gaussmom.gaussmom_core import (
+    GaussMomData,
+    GaussMomObs,
+    _eval_gauss2d,
+)
+
+
+@jax.jit
+def fwhm_to_sigma(fwhm: float):
+    """
+    convert fwhm to sigma for a gaussian
+    """
+    return fwhm / 2.3548200450309493  # sig = fwhm / sqrt(8 * ln 2)
+
+
+@jax.jit
+def fwhm_to_T(fwhm):
+    """
+    convert fwhm to T for a gaussian
+    """
+    sigma = fwhm_to_sigma(fwhm)
+    return 2 * sigma**2
```

Comment:
These should go into a separate moments module or similar since they will be useful in other places too.